docs/maps/import-geospatial-data.asciidoc

```diff
@@ -58,6 +58,54 @@ On the {kib} home page, you can upload a file and import it into an {es} index w
 . Use the file chooser to select a GeoJSON file.
 . Click *Import file*.
 
+[discrete]
+=== Draw features in map
+
+Upload features into Elasticsearch by drawing lines, polygons, circles, bounding boxes, and points in a map.
+
+To create a new index for drawing:
+
+. <<maps-create, Create a new map>>.
+. Click *Add layer*.
+. Select *Create index*.
+. Set *Index name*.
+. Click *Create index* button.
+
+To open an existing index for drawing:
+
+. <<maps-create, Create a new map>>.
+
+. Click *Add layer*.
+
+. Select *Documents*.
+
+. Select *Data view* that points to your index. A <<data-views, data view>> can point to one or more indices. For feature editing, the data view must point to a single index.
```

Comment:
Should this be one of the following?

- Select the data view that points to your index.
- In the **Data view** dropdown, select the data view that points to your index.